src/barril/units/_scalar.py

```diff
@@ -242,8 +242,8 @@ def GetFormatted(self, unit=None, value_format=None):
     def __eq__(self, other):
         return (
             type(self) is type(other)
-            and self._value == other.value
-            and self._quantity == other._quantity
+            and self._value == other.GetValue(self.unit)
```

Comment:
OMG

Reply:
What? Do you think I'll break everything? 🙏